src/sparse/impl/KokkosSparse_spmv_bsrmatrix_impl.hpp

```diff
@@ -516,23 +494,33 @@ struct BsrMatrixSpMVTensorCoreDispatcher {
   // to be used to avoid instantiating on unsupported types
   static void tag_dispatch(std::false_type, YScalar, AMatrix, XMatrix, YScalar,
                            YMatrix) {
-    KokkosKernels::Impl::throw_runtime_exception(
-        "unsupported for complex types");
+    KokkosKernels::Impl::throw_runtime_exception("unsupported for arguments");
```

Comment:
You could be a little more explicit about what is unsupported, it is clear reading the code but it will not be that clear when reading this out of context during a simulation.

Reply:
There's actually a run-time guard that should prevent this message from ever printing, but I will improve it all the same.